queries/diff/highlights.scm

```diff
@@ -13,3 +13,30 @@
 (location) @attribute
 
 (command) @function
+
+(context) @none
```

Comment:
I don't think that does _anything_. Can you try omitting it (or giving it a proper capture)?

Reply:
For sure; it indeed does nothing :laughing:

Reply:
(I know we have a bunch of uses still in our queries; they predate the Neovim highlighter, and some are due to my removal of `@text` -- I didn't want to delete the captures entirely, so I left them as placeholders.)

Reply:
Gotcha, that makes sense :+1:

Reply:
(Doesn't mean a cleanup -- where possible -- wouldn't be welcome ;))

Reply:
Any excuse to procrastinate my other work!